packages/ember-glimmer/lib/helpers/get.ts

```diff
@@ -79,8 +79,12 @@ class GetHelperReference extends CachedReference {
 
   static create(sourceReference: VersionedPathReference<Opaque>, pathReference: PathReference<string>) {
     if (isConst(pathReference)) {
-      let parts = pathReference.value().split('.');
-      return referenceFromParts(sourceReference, parts);
+      let value = pathReference.value();
+      if (typeof value === 'string' && value.indexOf('.') > -1) {
+        return referenceFromParts(sourceReference, value.split('.'));
```

Comment:
FWIW, I am working on a follow up PR (which will only target master) to make the `referenceFromParts` utility function accept a single value (instead of an array), that will remove this duplication (here and below)...